SMCollection/examples/example-normalized.json

```diff
@@ -28,5 +28,12 @@
   "isAnalyzedBy": {
```

Comment:
this is a format for NGSI-LD while this example is NGSIv2, such change should implemented in example-normalized.json**ld**

Reply:
for NGSIv2 it should be
"groupedAt":{
"type": "DateTime",
"value": "2020-12-24T12:00:00Z"
}